packages/components/src/icon/index.tsx

```diff
@@ -107,6 +107,8 @@ function Icon( {
 		return cloneElement( icon, {
 			// @ts-ignore Just forwarding the size prop along
 			size,
+			width: size,
+			height: size,
 			...additionalProps,
```

Comment:
This potentially generates invalid HTML or is a breaking change in cases like this:

```jsx
function MyIconDiv( { size, ...props } ) {
	return (
		<div style={ { width: size, height: size } } { ...props }>
			🏝️
		</div>
	);
}

function MyIconImg( { size, ...props } ) {
	return (
		<img
			width={ size + 1 }
			height={ size + 1 }
			src="https://s.w.org/style/images/about/WordPress-logotype-wmark.png"
			alt=""
			{ ...props }
		/>
	);
}

// Renders invalid attributes (width/height) on a div element
<Icon icon={ <MyIconDiv /> } />

// Overrides existing width/height values
<Icon icon={ <MyIconImg /> } />
```

A non-breaking fix would be preferable, but if not, we might want to note this as a breaking change.

Reply:
I updated the changelog and marked this as breaking.